src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -309,8 +310,24 @@ private function addWorkflowSection(ArrayNodeDefinition $rootNode)
                                         ->cannotBeEmpty()
                                     ->end()
                                     ->scalarNode('initial_place')
+                                        ->beforeNormalization()
+                                            ->ifTrue(function ($v) { return null !== $v; })
+                                            ->then(function ($v) {
+                                                @trigger_error('The "initial_place" configuration key has been deprecated in Symfony 4.3. Use the "initial_places" configuration key instead.', E_USER_DEPRECATED);
```

Comment:
there is native support for deprecating options now, can't we use it in this can?

Reply:
I looked at the file, and I did not see other example where we use this feature. Could you give me some example? Thanks.

Reply:
that's quite new so they aren't many, see eg `MainConfiguration` in `SecurityBundle`

Reply:
OK, I fixed it.
But I have an issue. We have `->fixXmlConfig('initial_place')` so Symfony already convert `initial_place` to `initial_places`.
Si I'm not able to trigger the deprecation :( We have to keep it to support XML...
If someone have a suggestion /cc @stof